compiler/rustc_target/src/spec/loongarch64_unknown_linux_gnu.rs

```diff
@@ -0,0 +1,17 @@
+use crate::spec::{Target, TargetOptions};
+
+pub fn target() -> Target {
+    Target {
+        llvm_target: "loongarch64-unknown-linux-gnu".into(),
+        pointer_width: 64,
+        data_layout: "e-m:e-p:64:64-i64:64-i128:128-n64-S128".into(),
+        arch: "loongarch64".into(),
+        options: TargetOptions {
+            cpu: "generic-la64".into(),
```

Comment:
Generic contains 32-bit or 64-bit, but we only added 64-bit support so far, so I think we still need to add  -la64

Reply:
> Generic contains 32-bit or 64-bit, but we only added 64-bit support so far, so I think we still need to add -la64

It's irrelevant, it's not like saying `generic` here would lead to LA32 codegen being enabled for `loongarch64` compilations, thanks to the specified triple. I'm only suggesting this to hopefully reduce the cognitive load for people unfamiliar with LoongArch internals.